packages/vaadin-crud/src/vaadin-crud.js

```diff
@@ -178,6 +178,10 @@ class CrudElement extends ElementMixin(ThemableMixin(PolymerElement)) {
           display: none !important;
         }
 
+        :host([no-toolbar]) [part='toolbar'] {
```

Comment:
Not a blocker: personally I recommend to move this after the `[part='toolbar']` block.
This way we will make the order of CSS overrides look more natural and easy to read.